src/java-opentimelineio/src/main/java/io/opentimeline/util/Pair.java

```diff
@@ -0,0 +1,11 @@
+package io.opentimeline.util;
+
+public class Pair<T, U> {
+    public T first;
```

Comment:
My recommendation is to use Java convention even for generic tools, by making variables private and use access/setter methods that can be easily generated from IntelliJ.